stories/topics/proc-gcp-backup-container-image.adoc

```diff
@@ -21,3 +21,8 @@ For more information about registry login, see link:https://access.redhat.com/Re
 $ export IMAGE={Registry}/ansible-on-clouds/{OpImageName}
 $ docker pull $IMAGE --platform=linux/amd64
 ----
+
+[NOTE]
+====
+If your foundation deployment version is not {ImageRef}-00, refer to the tables on the link:https://access.redhat.com/documentation/en-us/ansible_on_clouds/2.x/html/ansible-automation-platform-from-gcp-release-notes/assembly-gcp-release-notes-24[Released versions] page for a matching deployment version, in the column *Ansible on Clouds version*, and find the corresponding operational image to use, in the column *Ansible-on-clouds-ops container image*, for the IMAGE environment variable.
```

Comment:
in the Ansible on Clouds version column.

Find the corresponding....

Reply:
fixed